backend/src/config.ts

```diff
@@ -145,14 +158,23 @@ const defaults: IConfig = {
   },
   'SOCKS5PROXY': {
     'ENABLED': false,
+    'USE_ONION': true,
     'HOST': '127.0.0.1',
     'PORT': 9050,
     'USERNAME': '',
     'PASSWORD': ''
   },
   "PRICE_DATA_SERVER": {
     'TOR_URL': 'http://wizpriceje6q5tdrxkyiazsgu7irquiqjy2dptezqhrtu7l2qelqktid.onion/getAllMarketPrices',
-    'CLEARNET_URL': 'https://price.bisq.wiz.biz/getAllMarketPrices'
+    'CLEARNET_URL': 'https://price.bisq.wiz.biz/getAllMarketPrices',
+    'BISQ_URL': 'https://bisq.markets/api',
+    'BISQ_ONION': 'http://bisqmktse2cabavbr2xjq7xw3h6g5ottemo5rolfcwt6aly6tp5fdryd.onion/api'
```

Comment:
@ayanamidev shouldn't this be under `EXTERNAL_DATA_SERVER` as well?

Reply:
@knorrium You are right, it would make more sense to put them as `EXTERNAL_DATA_SERVER`